SPECS/memtest86+.spec

```diff
@@ -36,6 +36,9 @@ install -m644 memtest.{bin,efi} -D %{buildroot}/boot/
 /boot/memtest.bin
 
 %changelog
+* Thu Mar 02 2023 Yann Dirson <yann.dirson@vates.fr> - 6.01-3
```

Comment:
Rather than using a release number which doesn't correspond to any build, I simply write intermediate changelog this way:

```
* next
- stuff
- stuff
```